ledger/acctonline_test.go

```diff
@@ -1473,13 +1475,25 @@ func TestAcctOnlineTopBetweenCommitAndPostCommit(t *testing.T) {
 	_, totals, err := au.LatestTotals()
 	require.NoError(t, err)
 
+	// apply some rounds so the db round will make progress (not be 0)
+	i := 0
+	for ; i < 10; i++ {
+		var updates ledgercore.AccountDeltas
+		updates.Upsert(allAccts[numAccts-1].Addr, ledgercore.AccountData{
+			AccountBaseData: ledgercore.AccountBaseData{Status: basics.Offline}, VotingData: ledgercore.VotingData{}})
+		newBlockWithUpdates(genesisAccts, updates, totals, t, ml, i, oa)
+	}
+
+	stallingTracker.shouldLockPostCommit = true
+
 	updateAccountsRoutine := func() {
-		for i := 0; i < 10; i++ {
+		for ; i < 12; i++ {
```

Comment:
Idan, could comment the literal values here, above and below, like what is the rationale to have 10 blocks, not 3, not 20 added